packer/plugin-getter/plugins.go

```diff
@@ -280,6 +280,11 @@ func (pr Requirement) ListInstallations(opts ListInstallationsOptions) (InstallL
 			continue
 		}
 
+		if ver.Metadata() != "" {
```

Comment:
To confirm this case. On install a binary version that contains metadata will be converted to a final version, correct? 

If so then a user can not install using `packer init` or `packer plugins install` but if they drop the binary into a directory manually this change will prevent it from loading. Is that correct?

Reply:
The test in acc_test_logic indicate :point_up to be true but I want to confirm.

Reply:
That should be the case indeed, `packer plugins install` and `packer init` scrub the metadata part from a release/local's name, the `describe` command will still report it but ignore it for sorting purposes (as per semver), but then yes, if a plugin is installed with a `+metadata` in its version, we reject it explicitely.